sanicbot/core/cogs.py

```diff
@@ -0,0 +1,45 @@
+import aiohttp
+from discord.ext import commands
+from discord.ext.commands import Context
+
+from sanicbot.core.utils import failure_message, success_message, read_file
+
+
+class GitCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.org_url = f'https://github.com/sanic-org/'
+
+    async def lookup(self, number: int, repo):
```

Comment:
You should complete the annotation here.

Reply:
What do you mean?

Reply:
repo and return type

Reply:
Ah duh